modules/apps/layout/layout-reports-web/src/main/resources/META-INF/resources/js/components/error-alert/ErrorAlert.js

```diff
@@ -64,18 +64,18 @@ const ErrorAlert = () => {
 						pageCanNotBeAudited ? 'font-weight-bold' : undefined
 					}
 				>
-					{title}
+					{title}{' '}
 				</span>
 				{pageCanNotBeAudited && (
-					<span className="ml-1">
+					<span>
 						{Liferay.Language.get(
 							'page-is-not-accessible-from-the-internet-or-is-not-indexed'
-						)}
+						)}{' '}
```

Comment:
If our only option here is adding the space, maybe it would look a little bit better with string interpolation:
{'${Liferay.Language.get('you-can-set-the-api-key-from-site-settings-pages-google-pagespeed-insights')} '}.
What do you think? 😄

Reply:
I totally agree